src/common/components/KeypomLogo/KeypomLogo.tsx

```diff
@@ -0,0 +1,18 @@
+import { HStack, Box, Text } from "@chakra-ui/react";
+
+export const KeypomLogo = () => {
+  return (
+    <HStack spacing={{ base: 2, md: "2.5" }}>
+      <Box
+        bgColor="gray.800"
+        borderRadius="100%"
+        h={{ base: 5, md: "7" }}
+        rounded="full"
```

Comment:
is rounded still needed? there's already border-radius 100%

Reply:
yea. thanks for the catch. fixed it [here](https://github.com/keypom/keypom-airfoil/pull/11/commits/f0552eb49c41e6bd1b63a585b3456f7339e0e32f).